flixel/input/gamepad/FlxGamepadAnalogStick.hx

```diff
@@ -32,7 +32,7 @@ class FlxGamepadAnalogStick
 	/**
 	 * when analog inputs are received, how to process them digitally
 	 */
-	public var mode(default, null):FlxAnalogToDigitalMode = ONLY_ANALOG;
+	public var mode(default, null):FlxAnalogToDigitalMode = BOTH;
```

Comment:
It's not strictly speaking breaking; previously it was reporting only analog actions, this would make it report both analog and digital. As far as I can tell, the analog-to-digital actions weren't even mapped up correctly and thus weren't being reported. 

The only difference users should see is new digital actions being reported, and will only affect things if they choose to poll for them.